docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+
+#### File Types
+
+| Filename Pattern | Description                                                 | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram.crai | Cram Index file format. Requires a corresponding CRAM file. | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+## Quality Control Metrics
+
+Quality control metrics are collected and recorded at several checkpoints in the ARGO Analytics pipeline to ensure that only high-quality data is released. All QC Metrics are associated to a corresponding analysis/file set that they annotate.
+
+### Read Group QC
+
+file containing read group (lane) level QC metrics including --,--?.
+
+#### File Types
+
+| Filename Pattern       | Description                                    | Analysis Type | Data Category           | Generating Workflow(s) |
+| ---------------------- | ---------------------------------------------- | ------------- | ----------------------- | ---------------------- |
+| \*.ubam_qc_metrics.tgz | Generated by Picard CollectQualityYieldMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Alignment QC
+
+Files containing quality control metrics for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Duplicates Metrics
+
+Data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+
+#### File Types
+
+| Filename Pattern          | Description                                | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
```

Comment:
not sure, as above...seems like something is missing

Data file containing OxoG metrics, i.e. metrics quantifying the error rate resulting from oxidative artifacts for aligned CRAM files.

Reply:
rearranged to 2 sentences